### DISCUSSION TITLE
Can I use functional react components with react-joyride

### DISCUSSION THREAD

**Discussion by DionneChelini** [gilbarbara/react-joyride#863] [Q&A] [answered]
Hi there, am I able to use React Functional Components with react-joyride or do I need to use Class Components?

**Comment by shuvro111** [comment, marked as answer]
@DionneChelini Yes, you can. Here's an example with the functional components that react-joyride provides.
[Edit on Code Sandbox](https://codesandbox.io/s/github/gilbarbara/react-joyride-demo/tree/main)
